<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedShiftQtPrefs</class>
 <widget class="QDialog" name="RedShiftQtPrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>447</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxPath">
     <property name="title">
      <string>Redshift Path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditConf"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelConf">
        <property name="text">
         <string>Conf:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButtonConf">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelPath">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditPath"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButtonPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxLocalSettings">
        <property name="text">
         <string>Use RedShiftQt settings instead of redshift.conf</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lineEditConf</zorder>
     <zorder>labelConf</zorder>
     <zorder>toolButtonConf</zorder>
     <zorder>labelPath</zorder>
     <zorder>toolButtonPath</zorder>
     <zorder>checkBoxLocalSettings</zorder>
     <zorder>lineEditPath</zorder>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayoutLocalSettings">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxTemp">
       <property name="title">
        <string>Colour Temperature</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxTempTrans">
          <property name="text">
           <string>Smooth Transition</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelTempDay">
          <property name="text">
           <string>Day:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelTempNight">
          <property name="text">
           <string>Night:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxTempMethod">
          <item>
           <property name="text">
            <string>default</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelMethod">
          <property name="text">
           <string>Method:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxTempDay">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>3500</number>
          </property>
          <property name="maximum">
           <number>6500</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>6500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxTempNight">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>3500</number>
          </property>
          <property name="maximum">
           <number>6500</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBoxGam">
       <property name="title">
        <string>Screen Gamma</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="labelGamNight">
          <property name="text">
           <string>Night:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelGamDay">
          <property name="text">
           <string>Day:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamDayG">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamDayR">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamDayB">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QCheckBox" name="checkBoxGam">
          <property name="text">
           <string>Adjust Gamma</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamNightR">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamNightG">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGamNightB">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBoxBr">
       <property name="title">
        <string>Screen Brightness</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="labelBrDay">
          <property name="text">
           <string>Day:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelBrNight">
          <property name="text">
           <string>Night:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxBrNight">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxBrDay">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBoxBr">
          <property name="text">
           <string>Adjust Brightness</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBoxLoc">
       <property name="title">
        <string>User Location</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditLocLat">
          <property name="text">
           <string>00.00</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelLocProvider">
          <property name="text">
           <string>Provider:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxLocProvider">
          <item>
           <property name="text">
            <string>default</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelLocLon">
          <property name="text">
           <string>Longitude:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelLocLat">
          <property name="text">
           <string>Latitude:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditLocLon">
          <property name="text">
           <string>00.00</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonLocDetect">
          <property name="text">
           <string>Detect Location</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRun">
     <property name="text">
      <string>Start with RedShift enabled.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxShow">
     <property name="text">
      <string>Show this configuration dialog on application startup.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogButtons">
     <item>
      <widget class="QPushButton" name="pushButtonImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>62</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
